### DISCUSSION TITLE
How to rollback server version

### DISCUSSION THREAD

**Discussion by RootedGUIDance** [itzg/docker-minecraft-bedrock-server#299] [Q&A] [answered]
I have my server configured to get the LATEST version, and somehow my Windows 10 edition refuses to find the new client version to match the newly installed version for the server. Is there a way to rollback the bedrock server version to match my client without losing server data?

**Comment by itzg** [comment, marked as answer]
Set `VERSION` to the specific version you want to use. World data is left as is...assuming you're using a specific volume for `/data`

**Reply by RootedGUIDance** [reply]
Sorry for the ignorance, but what would that command look like for an existing container?

**Reply by itzg** [reply]
How do you have your container created so far?

**Reply by RootedGUIDance** [reply]
I believe I just created it using the quickstart (docker run -d -it -e EULA=TRUE -p 19132:19132/udp itzg/minecraft-bedrock-server) except I created a volume which I referenced with -v mc-volume:/data

**Reply by itzg** [reply]
The use of `-v` or not will make a big difference. You can determine what was done by using the `docker inspect` command. If you do have a specific volume, you'll stop and remove the container, and do the `run` again adding `-e VERSION=...`.

**Reply by RootedGUIDance** [reply]
Thanks so much!
